f2/src/forms/WhatWasAffectedForm.js

```diff
@@ -131,26 +111,28 @@ export const WhatWasAffectedForm = props => {
                             </Field>
                           </ConditionalForm>
                         )}
-                    </Box>
+                    </React.Fragment>
                   )
                 })}
-              </Stack>
-            </Control>
-
-            {showWarning ? (
-              <Control>
-                <Alert status="warning">
-                  <AlertIcon />
-                  <Trans id="whatWasAffectedForm.warning" />
-                </Alert>
-              </Control>
-            ) : null}
-            <NextAndCancelButtons
-              next={<Trans id="whatWasAffectedForm.nextPage" />}
-              button={<Trans id="whatWasAffectedForm.nextButton" />}
-            />
-          </Stack>
-        )}
+                {showWarning ? (
+                    <Control>
+                      <Alert status="warning">
+                        <AlertIcon />
+                        <Trans id="whatWasAffectedForm.warning" />
+                      </Alert>
+                    </Control>
+                ) : null}
+            </FormArrayControl>
+          </Control>
+          <P>
+            <Trans id="whatWasAffectedForm.expectations" />
```

Comment:
need translation strings for `whatWasAffectedForm.expectations`

Reply:
the translation string is now added